Create game

Wiyap opened the pull request.

GXcode95 committed:
Merge pull request #22 from GXcode95/form_components

Form components

Wiyap committed:
Creating CreateGame component

Wiyap committed:
Merge branch 'main' into create_game

GXcode95 commented on src/components/forms/CreateGame/SelectNumber.jsx:
```diff
@@ -0,0 +1,33 @@
+const SelectNumber = (props) => {
+  const numbers = Array.from({length: 100}, (x, i) => i+1)
+
+  function capitalizeFirstLetter(string) {
```
bien vu je te pique la méthode :P

GXcode95 committed:
indentation fix

GXcode95 committed:
Update index.jsx

Display  , as width, can be used directly as props, no needs to use the sx props

GXcode95 reviewed (approved):
as said in the last commit, prefer to use display and iwdth as props instead of using the `sx` props
